wgpu/src/backend/webgpu.rs

```diff
@@ -1562,6 +1562,40 @@ impl dispatch::InstanceInterface for ContextWebGpu {
         // Devices are automatically polled.
         true
     }
+
+    #[cfg(feature = "wgsl")]
+    fn wgsl_language_features(&self) -> crate::WGSLLanguageFeatures {
+        let mut wgsl_language_features = crate::WGSLLanguageFeatures::empty();
+        if let Some(gpu) = &self.gpu {
+            gpu.wgsl_language_features()
+                .keys()
+                .into_iter()
+                .map(|wlf| wlf.expect("WGSLLanguageFeatures elements should be valid"))
+                .map(|wlf| {
+                    wlf.as_string()
+                        .expect("WGSLLanguageFeatures should be string set")
```

Comment:
nitpick: It's nice to put backticks on code-ish things.

```suggestion
                .map(|wlf| wlf.expect("`WGSLLanguageFeatures` elements should be valid"))
                .map(|wlf| {
                    wlf.as_string()
                        .expect("`WGSLLanguageFeatures` should be string set")
```